src/mmw/apps/modeling/views.py

```diff
@@ -612,35 +478,33 @@ def drb_point_sources(request):
                     headers={'Cache-Control': 'max-age: 604800'})
 
 
-def start_celery_job(task_list, job_input, user=None):
-    """
-    Given a list of Celery tasks and it's input, starts a Celery async job with
-    those tasks, adds save_job_result and save_job_error handlers, and returns
-    the job's id which is used to query status and retrieve results via get_job
-
-    :param task_list: A list of Celery tasks to execute. Is made into a chain
-    :param job_input: Input to the first task, used in recording started jobs
-    :param user: The user requesting the job. Optional.
-    :return: A Response contianing the job id, marked as 'started'
-    """
-    created = now()
-    job = Job.objects.create(created_at=created, result='', error='',
-                             traceback='', user=user, status='started',
-                             model_input=job_input)
-
-    success = save_job_result.s(job.id, job_input)
-    error = save_job_error.s(job.id)
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+def get_job(request, job_uuid, format=None):
```

Comment:
This got reordered in the file, so the diff looks off. It should be implemented identically to before.